ebean-core/src/main/java/io/ebeaninternal/server/query/DbOrderByTrim.java

```diff
@@ -14,4 +14,21 @@ static String trim(String dbOrderBy) {
     return orderByTrim.matcher(dbOrderBy).replaceAll("");
   }
 
+  /**
+   * Checks, if <code>sql</code> contains the <code>column</code>.
+   * <p>
+   * SQL is normally comma separated: "t0.id, t1.id, t2.name"
+   */
+  static boolean contains(String sql, String column) {
+    if (sql.endsWith(column)) { // simplest way. sql ends with the column
+      return true;
+    } else if (sql.contains(column)) {
+      // We need to check, if it is really the correct column,
+      // sql="t0.name_short, t0.name_long" will match on column="t0.name"
+      return sql.contains(column + ",");
```

Comment:
Just pondering if we want to get the index position of column and then check the following char == ',' rather than perform the string contains twice (+ a string concat).

Otherwise looks great !!

Reply:
My comment about using index position is misplaced - this looks as good as we will get.

I'm happy here.